contracts/governance/AxelarMultisigMintLimiter.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { IMultisigExecute } from '../interfaces/IMultisigExecute.sol';
+import { MultisigBase } from '../auth/MultisigBase.sol';
+
+contract AxelarMultisigMintLimiter is MultisigBase, IMultisigExecute {
+    constructor(address[] memory accounts, uint256 threshold) {
+        _rotateSigners(accounts, threshold);
+    }
+
+    function execute(address target, bytes calldata callData) external onlySigners {
```

Comment:
add ether value that should be forwarded to the target as an arg. Makes it more generic, allows withdrawing ether for e.g. etc
```suggestion
    function execute(address target, bytes calldata args, uint value) external onlySigners {
```